services/API-service/src/scripts/seed-init.ts

```diff
@@ -179,11 +180,16 @@ export class SeedInit implements InterfaceScript {
     const seedRedcrossBranches = new SeedRedcrossBranches(this.connection);
     await seedRedcrossBranches.run();
 
-    // ***** SEED INDICATOR DATA PER ADMIN AREa *****
+    // ***** SEED INDICATOR DATA PER ADMIN AREA *****
     console.log('Seed Indicator data per admin-area...');
     const seedAdminAreaData = new SeedAdminAreaData(this.connection);
     await seedAdminAreaData.run();
 
+    // ***** SEED RAINFALL DATA *****
+    console.log('Seed rainfall data...');
+    const seedOther = new SeedRainfallData(this.connection);
```

Comment:
`seedOther` to `seedRainfall`

Reply:
done (when committed) > forgot this one in the previous editing round